scene/scene.js

```diff
@@ -114,17 +114,16 @@ optionButtonDiv.addEventListener('click', () => {
 
 
 continueButtonDiv.addEventListener('click', () => {
-
-    if (user.visited.length === 4) {
-        window.location.replace('../reult/index.html');
+    user.counter++;
+    setInLocalStorage('user', user);
+    alert('conitinue2222    user.counter: ' + user.counter);
+    if (user.counter === 4) {
```

Comment:
nice work, definitely can implement other factors as well to get to result page to try and make a more user-friendly app

Reply:
thank you